messaging-2/methods/createGroup.md

```diff
@@ -9,14 +9,14 @@ Send a group text message or picture message
 <code class="post">POST</code>`https://api.catapult.inetwork.com/v2/users/{userId}/messages`
 
 ### Supported Parameters
-| Parameter     | Type     | Description                                                                                                  | Mandatory |
-|:--------------|:---------|:-------------------------------------------------------------------------------------------------------------|:----------|
-| from          | `string` | One of your telephone numbers the message should come from (must be in E.164 format, like +19195551212).     | Yes       |
-| to            | `array`  | The phone number the message should be sent to (must be in E.164 format, like +19195551212).                 | Yes       |
-| text          | `string` | The contents of the text message (must be 2048 characters or less).                                          | Yes       |
-| applicationId | `string` | The ID of the Application your `from` number is associated with in the Bandwidth Phone Number Dashboard.     | Yes       |
-| media         | `array`  | A list of URLs to include as media attachments as part of the message.                                       | No        |
-| tag           | `string` | Any string which will be included in the callback events of the message.                                     | No        |
+| Parameter     | Type                             | Description                                                                                                  | Mandatory |
+|:--------------|:---------------------------------|:-------------------------------------------------------------------------------------------------------------|:----------|
+| from          | `string`                         | One of your telephone numbers the message should come from (must be in E.164 format, like +19195551212).     | Yes       |
+| to            | `string` or `array` of `string`  | The phone number or numbers the message should be sent to (must be in E.164 format, like +19195551212).      | Yes       |
```

Comment:
Is there anytime a group message **Won't be** an `array` of `strings`?

Reply:
no, a group message will always be an array